Documentos/AnalisisEstaticoCodigoBackEnd.md

```diff
@@ -283,6 +283,58 @@ Esto se ve corregido en otros tests, en los que se usa la anotación *DataTestMe
 [DataRow(GuestType.Baby, 6)]
 public void DiscountAppliesToAllBabies(GuestType guestType, int amountGuests) { ... }
 ````
+---
+**Comparación de strings usando `==`**
+
+A modo de ejemplo, en el siguiente método, dentro de la clase `JSONImporter`, podemos ver que se realiza la comparación de strings utilizando `==`, en vez de usando el método `.equals`. 
+
+```
+public List<ImportedResort> RetrieveResorts(List<ImportingParameterValue> parameters)
+        {
+            List<ImportedResort> parsedResorts = new List<ImportedResort>();
+
+            ValidateParametersOrFail(parameters);
+            string fileName = parameters.Find(p => p.Name == "File to parse").Value; 
+
+            try
+            {
+                ...
+```
+
+Por más que el operador `==` compara strings, usar `.equals` es más robusto ya que permite la comparación de strings ignorando si son en mayúsculas o minúsculas.
+
+---
+**Importación de múltiples archivos** 
```

Comment:
se pedian importar multiples archivos?

Reply:
no se, pero puede ser un _nice to have_, como que la solución sería más robusta